doc/man/8/ceph.rst

```diff
@@ -23,7 +23,7 @@ Synopsis
 
 | **ceph** **df** *{detail}*
 
-| **ceph** **fs** [ *ls* \| *new* \| *reset* \| *rm* \| *authorize* ] ...
+| **ceph** **fs** [ *add_data_pool* \| *authorize* \| *dump* \| *feature ls* \| *flag set* \| *get* \| *ls* \| *lsflags* \| *new* \| *rename* \| *reset* \| *required_client_features add* \| *required_client_features rm* \| *rm* \| *rm_data_pool* \| *set*] ...
```

Comment:
I see this is re-ordered alphabetical wise. Did you also adjust the descriptions for each sub-command below?

Reply:
Yes.

Reply:
OK - I couldn't see the reordering initially. Looks fine.